pyvcloud/vcd/org.py

```diff
@@ -373,3 +373,68 @@ def capture_vapp(self,
             rel=RelationType.ADD,
             media_type=EntityType.CAPTURE_VAPP_PARAMS.value,
             contents=contents)
+
+    def create_user(self, user_name, password, role_href, full_name, description, email, telephone, im, alert_email,
+                    alert_email_prefix, stored_vm_quota, deployed_vm_quota, is_group_role,
+                    is_default_cached, is_external, is_alert_enabled, is_enabled):
+        self.resource = self.client.get_resource(self.href_admin)
```

Comment:
only if resource is None

Reply:
Since  get_role sets resource to non-admin version of org. (when running user vcd-cli command)

def get_role(self, role_name):
        if self.resource is None:
            self.resource = self.client.get_resource(self.href)

I  need to force get the admin version of org. Create user api is 
https://bos1-vcd-sp-static-200-5.eng.vmware.com/api/admin/org/a93c9db9-7471-3192-8d09-a8f7eeda85f9/users (/api/admin/org instead of /api/org)

if I do the following in create_user()
 if self.resource is None:
            self.resource = self.client.get_resource(self.href)

I get an error as it is trying to create the user with uri /api/org/org_id/users instead of /api/admin/org/org_id/users

Reply:
In that case it might be better not to update the instance variable `self.resource` and just use a variable local to the `create_user` method, for example `resource_admin`

Reply:
changed it to resource_admin